### PR TITLE
Donator plush: BriareosBlue

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Donator reward for BriareosBlue

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl: LT3
image: A courier synth ready to deliver hugs and beeps straight to the heart!
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -645,3 +645,12 @@
 	attack_verb_simple = list("sigh")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/female/female_sigh.ogg' = 1)
 	gender = FEMALE
+
+//Donation reward for BriareosBlue
+/obj/item/toy/plush/skyrat/courier_synth
+	name = "courier synth plushie"
+	desc = "This synth plushie looks about ready to deliver hugs and beeps, straight to the heart! The tag seems to have an advertisement for a delivery company on it..."
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	desc = "This synth plushie looks about ready to deliver hugs and beeps straight to the heart! The tag seems to have an advertisement for a delivery company on it..."
```

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -645,3 +645,12 @@
...
+	icon_state = "plushie_courier"
+	attack_verb_continuous = list("delivers to", "export scans", "dwoops at", "bwuhs at", "stamps")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	attack_verb_continuous = list("delivers", "export scans", "dwoops", "bwuhs", "stamps")
```

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 3]
```diff
@@ -645,3 +645,12 @@
...
+	attack_verb_simple = list("deliver to", "export scan", "dwoop", "bwuh", "stamp")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	attack_verb_simple = list("deliver", "export scan", "dwoop", "bwuh", "stamp")
```

**Review by tf-4 — DISMISSED** [review]

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm [hunk 4]
```diff
@@ -260,6 +260,11 @@ GLOBAL_LIST_INIT(loadout_toys, generate_loadout_items(/datum/loadout_item/toys))
 /datum/loadout_item/toys/jecca
 	name = "Sexy Snoodle Plushie"
 	item_path = /obj/item/toy/plush/skyrat/jecca
+
+/datum/loadout_item/toys/courier_synth
+	name = "Courier Synth Plushie"
+	item_path = /obj/item/toy/plush/skyrat/courier_synth
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm:266]: They didn't want it ckey-whitelisted, then, I take it?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm (hunk 4 above)
Comment (lessthnthree) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm:266]: Correct.

**Comment by lessthnthree** [comment]
@Useroth

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**